module/module.c

```diff
@@ -661,42 +663,49 @@ static int handle_comment_data(merlin_node *node, merlin_header *hdr, void *buf)
 
 	/* make sure the object this comment is for exists */
 	if (!ds->service_description) {
-		if (!find_host(ds->host_name)) {
+		hs	= find_host(ds->host_name);
```

Comment:
Maybe there is no reason to use tab here?

Reply:
Agree, doesn't match current style of the code. Will update to reflect it.